README.md

```diff
@@ -178,7 +178,7 @@ cradle callPeak -ctrlbw ctrl1_corrected.bw ctrl2_corrected.bw ctrl3_corrected.bw
 
 ## Output file format
 ```
-chr	start	end	Name	score	strand	effectSize	inputCount	outputCount	-log(pvalue)	-log(qvalue)
+chr start   end Name    score   strand  effectSize  inputCount  outputCount -log(pvalue)    -log(qvalue)
```

Comment:
@Young-Sook I swapped spaces for tabs here because that's what is output by the program.